modulemd/v2/include/modulemd-2.0/private/modulemd-module-private.h

```diff
@@ -53,6 +54,31 @@ modulemd_module_set_defaults (ModulemdModule *self,
                               ModulemdDefaults *defaults);
 
 
+/**
+ * modulemd_module_associate_translation:
+ * @self: This #ModulemdModule object
+ * @translation: A #ModulemdTranslation object to associate with this
+ * #ModulemdModule
```

Comment:
GObject treats it as implicit here, but I'd prefer the `(in)` annotation for `@translation` as well as having the comment expressly state that it takes a reference and not a copy of the translation.